src/Pages/MatchResult/index.jsx

```diff
@@ -0,0 +1,86 @@
+import * as S from './style';
+import { useLocation, Link } from 'react-router-dom';
+import heart from '../../imgs/heart.png';
+import { useState } from 'react';
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { useState, useEffect } from 'react';
```

import 구문을 축약할 수 있겠네요

Reply:
수정 했습니다